lib/modules/userHighlight.js

```diff
@@ -256,6 +278,14 @@ module.go = () => {
 	if (module.options.highlightSelf.value && username) {
 		highlight(`author[href$="/${username}"]`, module.options.selfColor.value, module.options.selfColorHover.value);
 	}
+	if (module.options.highlightOpMentions.value) {
```

Comment:
Also restrict this to `isPageType('comments')` to that the users are not highlighted when browsing posts.